disnake/errors.py

```diff
@@ -29,18 +29,13 @@
 
 if TYPE_CHECKING:
     from aiohttp import ClientResponse, ClientWebSocketResponse
+    from requests import Response
```

Comment:
this won't always be installed in every environment.

Reply:
It definitely isn't, however pyright doesn't allow using variables in annotations, and as of pyright 1.1.250 this results in an error (second entry in [changelog](https://github.com/microsoft/pyright/releases/tag/1.1.250)). Perhaps you know of a better solution, although there's no point in having a try/except block inside `if TYPE_CHECKING:` regardless.